dpnp/backend/extensions/lapack/gesv_batch.cpp

```diff
@@ -56,6 +58,8 @@ static sycl::event gesv_batch_impl(sycl::queue &exec_q,
                                    const std::int64_t n,
                                    const std::int64_t nrhs,
                                    const std::int64_t batch_size,
+                                   [[maybe_unused]] const std::int64_t stride_a,
+                                   [[maybe_unused]] const std::int64_t stride_b,
```

Comment:
Can it be done in the way:
```suggestion
#if defined(USE_ONEMKL_INTERFACES)
                                   const std::int64_t stride_a,
                                   const std::int64_t stride_b,
#endif // USE_ONEMKL_INTERFACES
```